src/carbon_calculator/CCDefaults.py

```diff
@@ -119,15 +125,19 @@ def exportDefaults(self,fileName):
         if csvfile:
             csvfile.close()
         return status
+    
     def importDefaults(self,fileName):
         csvfile = None
+        print("Updating Carbon Calculator constant values.")
+        removeDuplicates()
         try:
             status = True
             with open(fileName, newline='') as csvfile:
                 inputlist = csv.reader(csvfile)
                 first = True
+                num = 0
                 for item in inputlist:
-                    if first:
+                    if first:   # header row
                         t = {}
```

Comment:
it is probably a better idea to not name variables with one letter.   It makes it hard to follow or even guess what it contains.

Also first is also weird.  Would you go for something like `is_first_run` ?

Reply:
done